tensorflow/lite/delegates/gpu/common/tasks/group_normalization_test_util.cc

```diff
@@ -0,0 +1,119 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow/lite/delegates/gpu/common/tasks/group_normalization_test_util.h"
+
+#include <memory>
+#include <vector>
+
+#include "tensorflow/lite/delegates/gpu/common/operations.h"
+#include "tensorflow/lite/delegates/gpu/common/status.h"
+#include "tensorflow/lite/delegates/gpu/common/task/testing_util.h"
+#include "tensorflow/lite/delegates/gpu/common/tasks/group_normalization.h"
+
+namespace tflite {
+namespace gpu {
+
+absl::Status GroupNormalizationTest(TestExecutionEnvironment* env,
+                              bool constant_idx) {
+  TensorFloat32 src_tensor;
```

Comment:
Could define this whole bunch of variables in a shorter way

Reply:
These tensor names are standard across all the tests, so it should be fine. The sample reference test :- 
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/delegates/gpu/common/tasks/pooling_test_util.cc